app/src/main/java/com/example/tmdb/ui/screens/DetailsScreen.kt

```diff
@@ -241,11 +241,32 @@ fun StarButton(
     )
 }
 
-/*@Preview
 @Composable
-fun DetailsScreenPreview() {
-    DetailsScreen(
-        navController = rememberNavController(),
-        movieId = 1
-    )
-}*/
+fun CrewRow(
+    modifier: Modifier = Modifier,
+    crew: List<CrewMember>
+) {
+    LazyRow(
+        modifier = modifier.padding(start = dimensionResource(id = R.dimen.padding_md))
+    ) {
+        for (i in 0 until crew.size-1 step 2) {
+            item {
+                Column(
+                    modifier = modifier.padding(end = dimensionResource(id = R.dimen.padding_sm))
+                ) {
+                    CrewMemberViewState(
+                        name = crew[i].name,
+                        job = crew[i].job
+                    )
+
+                    Spacer(modifier = modifier.height(dimensionResource(id = R.dimen.spacer_m)))
+
+                    CrewMemberViewState(
+                        name = crew[i + 1].name,
+                        job = crew[i + 1].job
+                    )
+                }
+            }
+        }
+    }
```

Comment:
Interesting solution, but unfortunately it will only work for list sizes of 4 to 6. It is not big deal so leave it as it is, you can change it later if you want.